README.rst

```diff
@@ -380,14 +380,47 @@ logins on the same device, you do not wish the old user to receive messages whil
 Via DRF, any creation of device with an already existing registration ID will be transformed into an update.
 If done manually, you are responsible for deleting the old device entry.
 
+Using custom FCMDevice model
+----------------------------
+
+If there's a need to store additional information or change type of fields in the FCMDevice model.
+You could simple override this model. It could be fully your own model or you could inheriting from AbstractFCMDevice 
```

Comment:
If they use fully their own model, how is that compatible with fcm-django's migrations?

Reply:
So it's a completely different model. 
* fcm-django's migrations migrate FCMDevice model
* other models migrations migrate other models

Reply:
So they have an empty FCMDevice table in their DB because of that? This should be at least documented

Reply:
But it's how does Django works during overriding `User` model. There's a package that use [swapper](https://pypi.org/project/django-cities/#swappable-models) and they don't copy paste Django or Swapper docs.

Reply:
I was not asking you to copy paste django or swapper docs, but to answer a common question that might arise from users in the documentation...

Reply:
> but to answer a common question that might arise from users in the documentation..

I updated doc with answers on questions that you raise